cluster/juju/layers/kubernetes-master/README.md

```diff
@@ -1,118 +1,102 @@
-# kubernetes
+# kubernetes-master
 
-[Kubernetes](https://github.com/kubernetes/kubernetes) is an open
-source system for managing application containers across multiple hosts.
-This version of Kubernetes uses [Docker](http://www.docker.io/) to package,
-instantiate and run containerized applications.
+[Kubernetes](http://kubernetes.io/) is an open source system for managing 
+application containers across a cluster of hosts. The Kubernetes project was
+started by Google in 2014, combining the experience of running production 
+workloads combined with best practices from the community.
 
-This charm is an encapsulation of the
-[Running Kubernetes locally via
-Docker](http://kubernetes.io/docs/getting-started-guides/docker)
-document. The released hyperkube image (`gcr.io/google_containers/hyperkube`)
-is currently pulled from a [Google owned container repository
-repository](https://cloud.google.com/container-registry/).  For this charm to
-work it will need access to the repository to `docker pull` the images.
+The Kubernetes project defines some new terms that may be unfamiliar to users
+or operators. For more information please refer to the concept guide in the 
+[getting started guide](http://kubernetes.io/docs/user-guide/#concept-guide).
 
-This charm was built from other charm layers using the reactive framework. The
-`layer:docker` is the base layer. For more information please read [Getting
-Started Developing charms](https://jujucharms.com/docs/devel/developer-getting-started)
+This charm is an encapsulation of the Kubernetes master processes and the 
+operations to run on any cloud for the entire lifecycle of the cluster.
 
-# Deployment
-The kubernetes charms require a relation to a distributed key value store
-(ETCD) which Kubernetes uses for persistent storage of all of its REST API
-objects.
-
-```
-juju deploy trusty/etcd
-juju deploy kubernetes-master
-juju deploy kubernetes-node
-juju add-relation kubernetes-master etcd
-juju add-relation kubernetes-node etcd
-```
+This charm was built from other charm layers using the Juju reactive framework.
+The other layers focus on specific subset of operations making this layer 
+specific to operations of Kubernetes master processes.
 
-# Configuration
-For your convenience this charm supports some configuration options to set up
-a Kubernetes cluster that works in your environment:  
+This charm is one part of what is needed to model a Kubernetes cluster. If you
+wish to deploy a complete cluster please look at the collection of charms that
+model a [Kubernetes core](https://jujucharms.com/kubernetes-core/) cluster.
 
-#### dns_domain
-
-The domain name to use for the Kubernetes cluster by the skydns service.
-
-#### enable-dashboard-addons
-
-Enables the installation of Kubernetes dashboard, Heapster, Grafana, and
-InfluxDB.
+# Deployment
 
-# Additional Information
+This charm is not self contained, other charms are needed to model a complete 
```

Comment:
I see the duplication now that you pointed it out. Not intentional, will re-factor.